src/views/AddItem.jsx

```diff
@@ -1,3 +1,93 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { addItem } from '../api/firebase';
+import { useState } from 'react';
+
+export function AddItem({ listId }) {
+	// declare itemName, daysUntilNextPurchase, and radioSelect
+	// into separate state variables
+	const [itemName, setItemName] = useState('');
+
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
+
+	const [radioSelect, setRadioSelect] = useState('soon');
+
+	// Store values inside itemData and sent over to database
+	// otherwise log an error if request fails
+	const onFormSubmit = (event) => {
+		event.preventDefault();
+		const itemData = {
+			itemName,
+			daysUntilNextPurchase,
+		};
+		try {
+			addItem(listId, itemData);
+			alert(`${itemData.itemName} was added to your list`);
```

Comment:
Consider how your message is displayed on different browsers:

(Chrome):
<img width="901" alt="Screen Shot 2023-01-20 at 12 56 12 PM" src="https://user-images.githubusercontent.com/44907238/213803881-df032c65-baf5-456d-b863-506b7a55bced.png">

(Safari):
<img width="915" alt="Screen Shot 2023-01-20 at 12 56 42 PM" src="https://user-images.githubusercontent.com/44907238/213803933-a99ea89a-ae65-4fd0-b82f-f5b4674aa8f8.png">

Is there a way to make it more user-friendly?

Reply:
@daredtech Good point.  We noticed other groups seemed to show a confirmation through another view page.  Any ideas for how to make it more uniform instead of in an alert box?

Reply:
@daredtech Thanks Volha! How would you approach this issue?

Reply:
@Bustamor, @arcan9  I would have a custom component that would be displayed for a short time somewhere close to the form/entry or a reserved space for it in the form, and once the item is successfully added - toggle to display that and toggle back after some time. 

Perhaps @mtfoley and @djtaylor8 have some suggestions as well?

The way you have it now totally works and meets AC, it is just a UX suggestion ;)

Reply:
As I mentioned above, my approach would be to use a library for notifications (e.g. `react-hot-toast` or something similar). You definitely can create a component, but something like this is lightweight and easy to use...

Reply:
@djtaylor8 omg! `react-hot-toast` is hot indeed! 🔥 i'll be adding it to my toolbox